### PR TITLE
Fix issue 178. Compose box behaves similarly on all modes.

### PR DESCRIPTION
**Pull request by reached-ankita**
Changes the behavior of compose box on stream, public and private message threads.

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@reached-ankita, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/).

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*
				
@reached-ankita, thanks for signing the CLA!

**Comment by Sam1301** [comment]
@reached-ankita Hi, you might want to close your old pr https://github.com/zulip/zulip-android/pull/193which addresses the same issue.
Here's a guide to do that https://help.github.com/articles/closing-a-pull-request/

**Comment by reached-ankita** [comment]
Thanks a lot @Sam1301 !! :D Done.

File: app/src/main/java/com/zulip/android/activities/ZulipActivity.java [hunk 1]
```diff
@@ -259,10 +259,31 @@ public void muteTopic(Message message) {
...
+            if(messageEt.getText().toString().equals(""))
+            {
+                displayChatBox(false);
+                displayFAB(true);
+
+            }
+        }
+
+        else if(chatBox.getVisibility()==View.VISIBLE && isCurrentModeStream()){
+            //check if messageEt is empty or not
```
Comment (brainwane) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: Hi! I see that line # 264 and line # 274 are the same comment; is that intentional?

Comment (brainwane) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: A common piece of advice on code comments is that we should not merely say what the code is doing, but why we have made that choice -- giving context about our design choices for future people reading the code who might not understand what bug we are trying to solve, or what problem we're trying to work around. In this case you might choose to leave a code comment on why it matters whether messageEt is empty. But I'm less familiar with the Android codebase here so I may be wrong in suggesting that it's useful or necessary to do that here -- I'll leave that to @kunall17's judgment.

Comment (reached-ankita) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: Oh Ok Yes you are right! I will make those changes

**Comment by brainwane** [comment]
Thanks for the update! https://zulip.readthedocs.io/en/latest/version-control.html#commit-messages will help you re-write your commit message and https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request will help you in case you need help rebasing.

**Comment by kunall17** [comment]
Nice, thank you for working on this!.. :+1: 
@timabbott LGTM

**Comment by timabbott** [comment]
Merged, thanks @reached-ankita!!

**Pull request closed without merging**